React/Modules/RCTStatusBarManager.m

```diff
@@ -89,6 +89,13 @@ - (void)applicationWillChangeStatusBarFrame:(NSNotification *)notification
   [self emitEvent:@"statusBarFrameWillChange" forNotification:notification];
 }
 
+RCT_EXPORT_METHOD(getHeight:(RCTResponseSenderBlock)callback)
+{
+  callback(@[@{
+    @"height": [NSNumber numberWithFloat:[[UIApplication sharedApplication] statusBarFrame].size.height],
```

Comment:
Can you write this as:

    @"height": @([[UIApplication sharedApplication] statusBarFrame].size.height),

Otherwise, looks OK.